ctypesgen/printer_python/printer.py

```diff
@@ -209,7 +209,7 @@ def print_module(self, module):
         self.file.write("from %s import *\n" % module)
 
     def print_constant(self, constant):
-        self.file.write("%s = %s" % (constant.name, constant.value.py_string(False)))
+        self.file.write("%s = %s  " % (constant.name, constant.value.py_string(False)))
         self.srcinfo(constant.src)
```

Comment:
Am I right in guessing that the code currently concatenates sourceinfo onto the assignment line?
I'm not much into the old codebase anymore, we've rewritten the python printer entirely, and especially newlines integration, see
https://github.com/pypdfium2-team/ctypesgen/blob/pypdfium2/src/ctypesgen/printer_python/printer.py
https://github.com/pypdfium2-team/ctypesgen/blob/pypdfium2/docs/dev_comments.md

Reply:
you guessed correctly